docs/book/src/reference/metrics.md

```diff
@@ -0,0 +1,76 @@
+## Metrics
+
+By default, controller-runtime builds a global prometheus registry and
+publishes a collection of performance metrics for each controller.
+
+### Protecting the Metrics
+
+These metrics are protected by [kube-auth-proxy](https://github.com/brancz/kube-rbac-proxy)
+by default.
+
+You will need to grant permissions to your Prometheus server so that it can
+scrape the protected metrics. To achieve that, you can create a `clusterRole` and a 
+`clusterRoleBinding` to bind to the service account that your Prometheus server uses.
```

Comment:
nits:
```suggestion
scrape the protected metrics. To achieve that, you can create a `ClusterRole` and a 
`ClusterRoleBinding` to bind to the service account that your Prometheus server uses.
```
following [Use upper camel case for API objects](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects)